internal/route/api/v1/repo/tree.go

```diff
@@ -0,0 +1,78 @@
+package repo
+
+import (
+	"fmt"
+
+	"github.com/gogs/git-module"
+
+	"gogs.io/gogs/internal/context"
+	"gogs.io/gogs/internal/db"
+)
+
+type repoGitTreeResponse struct {
```

Comment:
Sorry I just came up a better name for it 😝 
```suggestion
type repoGitTree struct {
```